### ISSUE TITLE
Add define() to the MockEnvironment

### ISSUE THREAD

**Issue by edhgoose** [php-mock/php-mock#12]
I'm falling foul of `something` somewhere that is causing me an utter nightmare.

TLDR is that adding a `define` function on the MockEnvironment solved my woes. I'll submit a PR shortly.

I'm trying to test a class which works with `time()`. I have 5 PHP Unit tests which each test a different part of this class. If useful to know, the class is part of a Symfony2 project.

Part of two of these five tests is to `sleep()` for a period of several days. The sleep function is returning immediately so it's definitely working. Good stuff.

If I run the tests individually, the tests all pass and the requisite amount of time I sleep in the test is reflected in the class under test.

However, if I run the tests as a whole (I'm running them in PHPStorm), it is entirely dependent on the ordering of the tests as to whether they will `sleep` correctly. 

I believe something is caching something and causing it to fall foul of Bug #68541. But I can't for love nor money figure out what. I've tried producing a minimal test case, but everything I produce seems to work as intended. I've tried modifying everything I can think of to make it work, but no such luck.

Anyway, I found that adding a `define` function to the `MockEnvironment` and getting that to call `define` on all the mocks works. So I'll submit a PR!

**Comment by malkusch** [comment]
I'm Sorry to hear that you have such issues. If using `define()` solves it, it very much sounds like you hit Bug #68541. Thank you for the PR. Anyways, I was waiting for something to finally release a version 1.
